tests/OSPSuite.Core.IntegrationTests/ModuleIntegrationTests.cs

```diff
@@ -231,4 +232,49 @@ public void should_have_kept_existing_container()
          lungCell.ShouldNotBeNull();
       }
    }
+
+   internal class When_constructing_a_simulation_based_on_an_invalid_parmaeter_overwritten_with_PV : concern_for_ModuleIntegration
+   {
+      [Observation]
+      public void should_be_able_to_construct_the_simulation()
+      {
+         _result.ValidationResult.ValidationState.ShouldBeEqualTo(ValidationState.Valid, _result.ValidationResult.Messages.Select(x=>x.Text).ToString("\n"));
+      }
+
+      protected override Func<ModuleHelperForSpecs, SimulationConfiguration> SimulationConfigurationBuilder()
+      {
```

Comment:
@PavelBal @Yuri05 This is the exampel that pavel created. I just want to get a failing test